storage/extent_store.go

```diff
@@ -335,23 +332,6 @@ func (s *ExtentStore) Write(extentID uint64, offset, size int64, data []byte, cr
 	return nil
 }
 
-func (s *ExtentStore) checkOffsetAndSize(extentID uint64, offset, size int64) error {
-	if IsTinyExtent(extentID) {
```

Comment:
> tinyExtent not checked in Extent.checkOffsetAndSize,why not keep ExtentStore and abondon Extent

Yes, you are right, it would be appropriate to delete Extent.checkOffsetAndSize

Reply:
@leonrayang This pr has beed updated, please review again.